installer/scripts/tue-install-impl

```diff
@@ -223,7 +223,25 @@ function tue-install-git
             cres=
         fi
 
-        res=$res$cres
+        res="$res $cres"
+        cd $mem_pwd
+    fi
+
+    if [ -n "$BRANCH" ]; #Cannot be combined with version-if because this one might not exist
```

Comment:
Where is BRANCH defined? I don't see any difference to version, except from the error checking.